### PR TITLE
Home work9

### PR DESCRIPTION
**Pull request by KozyrickiS**

### CODE REVIEW

File: HW9_Music/ConsoleMusic/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            var output3 = JsonConvert.SerializeObject(itemBySong, Formatting.None);
+            Console.WriteLine(output3);
+
+            var itemByTitle = Song.GetSongDataByTitle("Creep");
+            Console.WriteLine(itemByTitle);
+            var output4 = JsonConvert.SerializeObject(itemByTitle, Formatting.Indented);
+            Console.WriteLine(output4);
+
+            Console.ReadKey();
+
```
Comment (VadzimPapko) [inline HW9_Music/ConsoleMusic/Program.cs:43]: just empty unused line

File: HW9_Music/ConsoleMusic/Song.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleMusic
+{
+    class Song
+    {
+        public string title { get; set; }
```
Comment (VadzimPapko) [inline HW9_Music/ConsoleMusic/Song.cs:11]: C# properties should be started with Upper case letter
public string title { get; set; } => `public string Title { get; set; }`

File: HW9_Music/ConsoleMusic/Song.cs [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        public TimeSpan duration { get; set; }
+        public string composer { get; set; }
+        public DateTime releaseDate { get; set; }
+        public Genre genre { get; set; }
+        public Song()
+        { }
```
Comment (VadzimPapko) [inline HW9_Music/ConsoleMusic/Song.cs:17]: you can delete empty default Ctor

File: HW9_Music/ConsoleMusic/Song.cs [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+                AlbumYear = song.releaseDate.Year,
+                SongGenre = song.genre
+            };
+            return res;
+        }
+
+        public static dynamic GetSongDataByTitle(string title)
+        {
+            Song song = new Song();
+            List<Song> database = DBSongs();
```
Comment (VadzimPapko) [inline HW9_Music/ConsoleMusic/Song.cs:49]: good stuff, I see a List collection

File: HW9_Music/ConsoleMusic/Song.cs [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+            Song song8 = new Song("No Surprises", 3, "Radiohead", 1997, Genre.Rock);
+            songs.Add(song8);
+            Song song9 = new Song("Creep", 3, "Radiohead", 1993, Genre.Rock);
+            songs.Add(song9);
+            Song song10 = new Song("T.N.T.", 3, "AC/DC", 1975, Genre.Rock);
+            songs.Add(song10);
+
+            return songs;
+        }
+
```
Comment (VadzimPapko) [inline HW9_Music/ConsoleMusic/Song.cs:96]: the same

File: HW9_Music/ConsoleMusic/Song.cs [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+                Title = song.title,
+                Composer = song.composer,
+                SongSeconds = song.duration.Seconds,
+                AlbumYear = song.releaseDate.Year,
+                SongGenre = song.genre
+            };
+            return res;
+        }
+
+        private static List<Song> DBSongs()
```
Comment (VadzimPapko) [inline HW9_Music/ConsoleMusic/Song.cs:68]: good solution to have static data instead of DB.

**Review by VadzimPapko — APPROVED** [review]
done

File: HW9_Music/ConsoleMusic/Program.cs (hunk 1 above)
Comment (KozyrickiS) [reply HW9_Music/ConsoleMusic/Program.cs:43]: empty line - deleted

File: HW9_Music/ConsoleMusic/Song.cs (hunk 2 above)
Comment (KozyrickiS) [reply HW9_Music/ConsoleMusic/Song.cs:11]: got it. fixed

File: HW9_Music/ConsoleMusic/Song.cs (hunk 3 above)
Comment (KozyrickiS) [reply HW9_Music/ConsoleMusic/Song.cs:17]: empty constructor deleted

File: HW9_Music/ConsoleMusic/Song.cs (hunk 4 above)
Comment (KozyrickiS) [reply HW9_Music/ConsoleMusic/Song.cs:49]: thanks)

File: HW9_Music/ConsoleMusic/Song.cs (hunk 5 above)
Comment (KozyrickiS) [reply HW9_Music/ConsoleMusic/Song.cs:96]: empty line deleted

**Pull request merged**